week-1/I-floats/exercise.js

```diff
@@ -1,2 +1,9 @@
 var numberOfStudents = 15;
 var numberOfMentors = 8;
+var sum = 15 + 8;
```

Comment:
This should be `var sum = numberOfStudents + numberOfMentors;` so that if the values of `numberOfStudents` or `numberOfMentors` changes then you do not need to update `var sum =`